dependencies.gradle

```diff
@@ -1,7 +1,7 @@
 // Add your dependencies here
 
 dependencies {
-    compile("com.github.GTNewHorizons:CodeChickenLib:1.1.5.3:dev")
-    compile("com.github.GTNewHorizons:CodeChickenCore:1.1.3:dev")
-    shadowImplementation("org.apache.commons:commons-csv:1.5")
+    compile('com.github.GTNewHorizons:CodeChickenLib:1.1.5.3:dev')
+    compile('com.github.GTNewHorizons:CodeChickenCore:1.1.3:dev')
+    shadowImplementation('org.apache.commons:commons-csv:1.8')
```

Comment:
Did you test this change?

Reply:
Yes, it builds, runs, and does it's thing.

1.9.0 blows up btw not sure why, blows up in completely unrelated areas.